components/molecules/close-contact-warning.tsx

```diff
@@ -1,75 +1,34 @@
 import React, {FC} from 'react';
-import {
-  StyleSheet,
-  TouchableWithoutFeedback,
-  View,
-  Text,
-  Image
-} from 'react-native';
+import {StyleSheet, Text} from 'react-native';
 import {useTranslation} from 'react-i18next';
 import {useNavigation} from '@react-navigation/native';
 
-import {SingleRow} from '../atoms/layout';
-
 import {colors} from '../../constants/colors';
-
-import {shadows, text} from '../../theme';
+import {text} from '../../theme';
+import {Card} from '../atoms/card';
 
 export const CloseContactWarning: FC = () => {
   const {t} = useTranslation();
   const navigation = useNavigation();
 
   return (
-    <TouchableWithoutFeedback
+    <Card
+      padding={{v: 0, h: 0}}
```

Comment:
I had to set the vertical padding to zero, even though the one set in `styles.card` was `16`,  to match the original design:

![original-2](https://user-images.githubusercontent.com/1233582/89636705-52e47d00-d8a9-11ea-9f48-fe515797eb1e.png)

With vertical padding equal to 16, it would be:

![vertical padding 16](https://user-images.githubusercontent.com/1233582/89636743-61329900-d8a9-11ea-9803-f08bb093ff99.png)